### ISSUE TITLE
Update new variable names in exo.py 

### ISSUE THREAD

**Issue by rbales-usgs** [USGS-CMG/stglib#56]
Exo variable names were changed in the new KOR software. Updates were made to the functions `read_exo` and `read_exo_header` to accommodate for these variable name changes. `Ds_rename_vars` was not updated and it is limiting QA/QC for those variables with name changes and excluding attributes in the .nc file. 

 After line 291 (in exo.py), I suggest adding `"Chlorophyll_ug_per_L": "Fch_906", "BGA_PE_RFU": "BGAPErfu", "BGA_PE_ug_per_L": "BGAPE",`

Thank you!

**Comment by dnowacki-usgs** [comment]
Thanks Bo! Please feel free to open a PR to make these changes. We should also implement a test for processing some sample exo files. Could you attach a file that failed using the current version?

**Comment by rbales-usgs** [comment]
Hey Dan, 
I shared an exo.csv file with you on teams (along with the globalatts and config files). If you process the exo.csv to .nc, you will notice that some attributes are excluded from BGA ug/L, BGA RFU, and Chl ug/L. Similarly, if you try any qaqc on those variables using features in the .yaml file, it will not work. 

Thanks for the help!

p.s. Let me know if there is a better way of sharing files on GitHub. The file I tried sending was not supported, here.
